tests/rule_based_profiler/test_rule_based_profiler.py

```diff
@@ -1084,6 +1099,7 @@ def test_get_profiler_with_too_many_args_raises_error(
 
 
 @mock.patch("great_expectations.data_context.data_context.BaseDataContext")
+@pytest.mark.integration
```

Comment:
Since data context is mocked, I think this can be treated as a unit test, thoughts?

Reply:
I agree! Changed